frontend/packages/dev-console/src/utils/resource-label-utils.ts

```diff
@@ -8,6 +8,14 @@ export const getAppLabels = (name: string, application: string, imageStreamName?
   };
 };
 
+export const getAppAnnotations = (gitURL: string, gitRef: string) => {
```

Comment:
please change `gitURL` to `gitUrl`

Reply:
Actually console follows the convention of uppercase acronyms so `gitURL` is correct`.